data/tokenizer.py

```diff
@@ -120,3 +128,75 @@ def untokenize(self, tokens: list[str]) -> pd.DataFrame:
         df = pd.DataFrame(velocities, columns=["velocity"])
 
         return df
+
+
+class DstartEncoder(MidiEncoder):
+    def __init__(self, bins: int = 200):
```

Comment:
```suggestion
    def __init__(self, n_bins: int = 200):
```

`bins` suggests something else